Bangazon-Tinkr/Controllers/RubbishController.cs

```diff
@@ -41,6 +45,21 @@ public IActionResult GetAllRubbish()
 
             return Ok(allRubbish);
         }
+        // api/Rubbish/2/createRubbish
+        [HttpPost("{userId}/createRubbish")]
+        public IActionResult CreateNewRubbish(int userId, Rubbish rubbishToAdd)
+        {
+            var user =  _usersRepository.GetUserById(userId);
```

Comment:
If the above changes are made you can use something like rubbishToAdd.userId to check for valid user.